src/augment/augment.js

```diff
@@ -46,6 +46,11 @@ export const makeAugmentedExecutableSchema = ({
     // Otherwise parse the SDL and get its definitions
     definitions = parse(typeDefs).definitions;
   }
+
+  definitions[0].directives = definitions[0].directives?.filter(

```

Comment:
Is definitions[0] always defined? I imagine there could be a case where it is not

Reply:
probably, but good point!